ga4gh/backend.py

```diff
@@ -23,14 +24,9 @@ class AbstractBackend(object):
     This class provides methods for all of the GA4GH protocol end points.
     """
     def __init__(self):
-        self._variantSetIdMap = {}
-        self._variantSetIds = []
-        self._referenceSetIdMap = {}
-        self._referenceSetIds = []
-        self._readGroupSetIdMap = {}
-        self._readGroupSetIds = []
-        self._callSetIdMap = {}
-        self._callSetIds = []
+        self._variantSetIdMap = collections.OrderedDict()
+        self._referenceSetIdMap = collections.OrderedDict()
+        self._readGroupSetIdMap = collections.OrderedDict()
```

Comment:
I can change back to the old method if you think it's important --- might bring the size of the PR down a bit too. Your call.